### ISSUE TITLE
[NEW FEATURE]-Chain Casting

### ISSUE THREAD

**Issue by patrickmohrmann** [patrickmohrmann/earthdawn4eV2#1169]
It shall be possible to select a spell to be cast with the chain casting option.

this requires a new system setting "optional rule - chain casting".

if this setting is active spells can be selected to be a chain casting spell inside of the matrix (somewhere in the Spells tab .- maybe a check mark in the matrix item...)

the option will be grayed out if the spell does not qualify for chain casting (GMs are able to select it anyway) - this should be indicated by a tooltip.

so basically spells without thread, or with a thread in an enhanced matrix, armored matrix (basically with enough threads pre woven to cast the spell)

if a spell is checked the effect of the spell shall automatically be added to the target and get a specific bonus tag somewhere (chain casting) 

how to select a target for the chaincasting ???
